torchmultimodal/modules/losses/albef.py

```diff
@@ -0,0 +1,216 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+from typing import Callable, Optional
+
+import torch
+import torch.nn.functional as F
+from torch import nn, Tensor
+from torch.nn import CrossEntropyLoss
+
+
+class ImageTextContrastiveLoss(nn.Module):
+    """
+    Compute the image-text contrastive loss from image-text similarity, as used in ALBEF.
```

Comment:
Would add the additional information that this supports the use of pseudo-targets and so is different from your vanilla image/text contrastive loss. Similar to with MLM, when alpha = 0.0 I think this should just reduce to standard contrastive loss. You can think about making the momentum similarities optional to support that case, and if so it's worth mentioning that as well here.